lib/vrl/stdlib/Cargo.toml

```diff
@@ -20,6 +20,7 @@ hostname = { version = "0.3", optional = true }
 lazy_static = { version = "1", optional = true }
 md-5 = { version = "0.9", optional = true }
 nom = { version = "6", optional = true }
+pnet = { version = "0.27", optional = true }
```

Comment:
Based on the changes to `Cargo.lock`, this seems to have pulled in a *LOT* of additional dependencies. Is there any other crate that could provide the features needed for this?

Reply:
Yeah, this crate does do a lot more than we strictly need. I was able to swap `pnet` for `pnet_datalink` which reduces the number of dependencies substantially. That crate still does more than we need, but at least doesn't pull in too much.

I haven't come across another crate that has robust cross-platform support for this.